### PR TITLE
Fix some style to make it closer to the original

### PR DESCRIPTION
**Pull request by stanley30543**

### CODE REVIEW

File: client/sidebar.jsx [hunk 1]
```diff
@@ -28,6 +28,15 @@ class Sidebar extends React.Component {
       .catch(() => null);
   }
 
+  categoryStyleBrand() {
+    const text = `${this.state.product.category} > ${this.state.product.style} > ${this.state.product.brand}`;
```
Comment (kzlsakal) [inline client/sidebar.jsx]: This line seems longer than 80 characters.  Could you use the following pattern to make use of multiple lines?

```
variable = `text`
  + `more text`
```

**Review by kzlsakal — CHANGES_REQUESTED** [review]
Hi Stanley, this looks good with only one improvement.

File: client/sidebar.jsx (hunk 1 above)
Comment (stanley30543) [reply client/sidebar.jsx]: Fixed (:D)

**Review by kzlsakal — APPROVED** [review]
looks great!

**Pull request merged**